omnibus-sepolia.toml

```diff
@@ -132,7 +141,292 @@ func = "setPoolConfiguration"
 args = [
     "<%= settings.sc_pool_id %>",
     [
-        { marketId = "<%= imports.legacyMarket.settings.marketId %>", weightD18 = 99, maxDebtShareValueD18 = "<%= parseEther('1') %>" },
+        { marketId = "<%= imports.legacyMarket.settings.marketId %>", weightD18 = 90, maxDebtShareValueD18 = "<%= parseEther('1') %>" },
```

Comment:
I cant remember exactly how `maxDebtShareValueD18` works.. Maybe some core people know, @dbeal-eth?
But I think if its set to 1, then weight is a percent of how the poolds liquidity is delegated to the defferent market it backs. See the sepolia config:
https://github.com/Synthetixio/synthetix-deployments/blob/main/omnibus-sepolia.toml#L171

Reply:
Hey @0xjocke yeah i understand, but i thought it would be like something along the line:
- Legacy Market 70%
- BFP 30%
I think we can hold a meeting with core-people if it's not clear the config on this?